swupd.zsh

```diff
@@ -157,7 +157,7 @@ if [[ -n "$state" ]]; then
         update)
           local -a updates; updates=(
             $global_opts
-            '(help status -d --download )'{-d,--download}'[Download all content, but do not actually install the update]'
+            '(help status -d --download )--download[Download all content, but do not actually install the update]'
```

Comment:
I think you forgot to remove the '-d' from here: `(help status -d --download )`

Reply:
Fixed. Thanks.